actions/mail/feedback.ts

```diff
@@ -14,4 +15,16 @@ export const sendNewFeedbackEmail = async (controller: User, feedback: Feedback)
         subject: "New Feedback Released",
         html,
     });
+}
+
+export const sendFeedbackSubmitter = async (pilot: User, feedback: Feedback) => {
```

Comment:
Could we potentially make this more descriptive? Is it just used for sending staff comments back to whoever submitted it or will it be used for more?